testing/go/create_table_test.go

```diff
@@ -159,3 +159,63 @@ func TestCreateTable(t *testing.T) {
 		},
 	})
 }
+
+func TestCreateTableInherit(t *testing.T) {
+	RunScripts(t, []ScriptTest{
+		{
+			Name: "Create table with inheritance",
+			SetUpScript: []string{
+				"create table t1 (a int);",
+				"create table t2 (b int);",
+				"create table t3 (c int);",
+				"create table t11 (a int);",
+			},
+			Assertions: []ScriptTestAssertion{
+				{
+					Query:    "create table t4 (d int) inherits (t1, t2, t3);",
+					Expected: []sql.Row{},
+				},
+				{
+					Query:    "insert into t4 values (1, 2, 3, 4);",
+					Expected: []sql.Row{},
+				},
+				{
+					Query: "select * from t4;",
+					Expected: []sql.Row{
+						{1, 2, 3, 4},
+					},
+				},
+
+				{
+					Query:    "create table t111 () inherits (t1, t11);",
```

Comment:
kinda seems like this should be illegal, it results in a table with two columns of the same name

Reply:
Would add a check for this in GMS, make sure it's case-insensitive

Reply:
Disregard, didn't realize the merge behavior

In the case, make sure the merge behavior is case-insensitive